lib/include/openturns/OTtestcode.hxx

```diff
@@ -379,6 +380,17 @@ inline void assert_almost_equal(const SymmetricMatrix &a, const SymmetricMatrix
   }
 }
 
+inline void assert_almost_equal(const Distribution &a, const Distribution &b, const Scalar rtol = 1.0e-5, const Scalar atol = 1.0e-8, const String errMsg = "")
+{
+  if (a.getImplementation()->getClassName() != b.getImplementation()->getClassName())
+    throw InvalidArgumentException(HERE) << "A and B must be the same distribution. A is a " << a.getImplementation()->getClassName() << " whereas B is a " << b.getImplementation()->getClassName();
+
+  if (a.getParameterDimension() != b.getParameterDimension())
```

Comment:
Wrong. A Beta distribution  (4 parameters) may be equal to a Uniform distribution (2 parameters)

Reply:
True, but is it a problem for unit tests? If you want, I could change the name of the function to make clear it is not about equality in a mathematical sense, but in an object sense.